database/migrations/000016_artifact_versions.up.sql

```diff
@@ -0,0 +1,15 @@
+-- Copyright 2024 Stacklok, Inc
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+DROP TABLE IF EXISTS artifact_versions CASCADE;
```

Comment:
We can't really roll this back easily / cleanly with the `.down.sql`.  We might want to omit this from the PR, and do this migration in the future once we've rolled this change out to production.

Reply:
good point, I wasn't sure about this and that why I [asked explicitly](https://github.com/stacklok/minder/pull/2235#discussion_r1473026955) and exactly the kind of issues I wanted you to catch.